js/ngl/writer.js

```diff
@@ -85,7 +85,7 @@ NGL.PdbWriter = function( structure, params ){
                 // Alignment of one-letter atom name such as C starts at column 14,
                 // while two-letter atom name such as FE starts at column 13.
                 var atomname = a.atomname;
-                if( atomname.length === 1 ) atomname = " " + atomname;
+                atomname = " " + atomname;
```

Comment:
In my opinion, you chose the wrong starting position for the atom names. If you start all from the same point (removing the if cause), one space is missing according to the RCSB pdb file format.